share/smack/top.py

```diff
@@ -537,6 +542,14 @@ def arguments():
                                 choices=['z3', 'cvc4', "yices2"], default='z3',
                                 help='back-end SMT solver')
 
+    verifier_group.add_argument(
+        '--portfolio-config',
+        metavar='FILE',
+        default=None,
+        action=FileAction,
+        type=str,
+        help='read portfolio thread specs from a .yaml')
```

Comment:
Change this help string to `read portfolio configuration in YAML format from FILE`